lib/perl/Genome/WorkflowBuilder/Command.pm

```diff
@@ -247,6 +247,11 @@ sub operation_type_attributes {
             $attributes{$expected_attributes{$name}} = $value;
         }
     }
+
+    unless (defined $attributes{'lsfQueue'}) {
+        $attributes{'lsfQueue'} = Genome::Config::get('lsf_queue_build_worker');
+    }
```

Comment:
At any rate, it looks like changing this default will force me to update the expected output of almost twenty tests (which highlights plenty of cases not setting defaults!), so I think I'll pull this change into its own PR.